modules/order/src/AdjustableInterface.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Drupal\commerce_order;
+
+/**
+ * Defines an interface for objects that interact with Adjustments.
+ *
+ * @see \Drupal\commerce_order\Entity\Order
+ * @see \Drupal\commerce_order\Entity\LineItem
+ */
+interface AdjustableInterface {
+
+  /**
+   * Adds an adjustment to the order.
+   *
+   * @param \Drupal\commerce_order\Adjustment $adjustment
+   *   The adjustment.
+   *
+   * @return $this
+   */
+  public function addAdjustment(Adjustment $adjustment);
```

Comment:
The correct order should probably be get / set / add / remove, no? We also usually have a has method as well, dunno if needed here.